src/locale/locales/de/messages.po

```diff
@@ -640,19 +626,19 @@ msgstr "Mein Konto löschen"
 
 #: src/view/screens/Settings.tsx:632
 msgid "Delete my account…"
-msgstr "Mein Konto löschen..."
+msgstr "Mein Konto löschen…"
 
 #: src/view/com/util/forms/PostDropdownBtn.tsx:214
 msgid "Delete post"
-msgstr "Posten löschen"
+msgstr "Beitrag löschen"
 
 #: src/view/com/util/forms/PostDropdownBtn.tsx:218
 msgid "Delete this post?"
-msgstr "Diesen Posten löschen?"
+msgstr "Diesen Beitrag löschen?"
 
 #: src/view/com/post-thread/PostThread.tsx:242
 msgid "Deleted post."
-msgstr "Posten gelöscht."
+msgstr "Gelöschter Beitrag."
```

Comment:
If this corresponds to the message after you deleted a post, it should be "Beitrag gelöscht."